.cargo/config

```diff
@@ -11,4 +11,3 @@ rustflags = ["-C", "target-feature=+crt-static"]
 
 [unstable]
 bindeps = true
-sparse-registry = true
```

Comment:
What does this do? Can you document the reason for this change?

Reply:
This is due to a bug with `cargo publish` when using sparse registry support. This might be resolved in the latest rust nightly toolchain version, but untested.

Reply:
Please make a separate commit for this change with motivation in the (longer) commit description